sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.aest.controller.manualReassessment.e2e-spec.ts

```diff
@@ -166,11 +165,12 @@ describe("AssessmentAESTController(e2e)-manualReassessment", () => {
         .expect(HttpStatus.UNPROCESSABLE_ENTITY)
         .expect({
           statusCode: HttpStatus.UNPROCESSABLE_ENTITY,
-          message: `Application cannot have manual reassessment in any of the statuses: ${ApplicationStatus.Cancelled}, ${ApplicationStatus.Overwritten} or ${ApplicationStatus.Draft}.`,
```

Comment:
Why is `${ApplicationStatus.Overwritten}` removed from the list of invalid status?